Cordis h2020 to graph

russwinch opened the pull request:
First draft of the OGM is created along with some code to move data from SQL to the graph.

A configuration file is required for neo4j, in the usual place containing:
- user
- password
- host

Outstanding work and problems are issues in the project.

Commit:
First version of H2020 ogm.

Most nodes are added for H2020.

jaklinger reviewed (approved):
Accepting the review on the basis that I'm going to dev it anyway. All looks good thus far!

jaklinger commented:
ps. I'm just rerunning the tests since it's been a month or so. I'll merge once done

jaklinger commented:
Sorry to crash the party @russwinch but this test failed.

Commit:
Add nesta repo to requirements

External repositories are now excluded from Travis requirements.

Travis build was failing on the nesta repository requirement.
If packages in here start to depend on nesta packages then they will
need to be patched, or this requirement removed.

Requirements updated with dependencies.

Commit:
Create basic package to transfer sql to graph.

Nodes are currently created manually, so some kind of schema between
sql and the graph needs to be implemented.
